pkg/controller/traceflow/controller_test.go

```diff
@@ -100,6 +100,8 @@ func TestTraceflow(t *testing.T) {
 		}
 
 		tfc.kubeClient.CoreV1().Pods("ns1").Create(context.TODO(), &pod1, metav1.CreateOptions{})
+		createdPod, _ := tfc.waitForPodInNamespace("ns1", "pod1", time.Second)
+		assert.NotNil(t, createdPod)
 		tfc.client.CrdV1alpha1().Traceflows().Create(context.TODO(), &tf1, metav1.CreateOptions{})
 		res, _ := tfc.waitForTraceflow("tf1", crdv1alpha1.Running, time.Second)
 		assert.NotNil(t, res)
```

Comment:
```suggestion
		require.NotNil(t, res)
```
Otherwise the following code would still panic when the test fails. `require` skips the following code if it fails.

Reply:
Thanks. Committed these changes.